tests/integration/seccomp.bats

```diff
@@ -33,3 +33,71 @@ function teardown() {
 	runc run test_busybox
 	[ "$status" -eq 0 ]
 }
+
+# TODO:
+# - Test other actions like SCMP_ACT_TRAP, SCMP_ACT_TRACE, SCMP_ACT_LOG.
+# - Test args (index, value, valueTwo, etc).
+
+@test "runc run [seccomp] (SCMP_ACT_ERRNO default)" {
+	update_config '.process.args = ["/bin/sh", "-c", "mkdir /dev/shm/foo"] |
+		.process.noNewPrivileges = false |
+		.linux.seccomp = {
+			"defaultAction":"SCMP_ACT_ALLOW",
+			"architectures":["SCMP_ARCH_X86","SCMP_ARCH_X32"],
+			"syscalls":[{"names":["mkdir"], "action":"SCMP_ACT_ERRNO"}]
+		}'
+
+	runc run test_busybox
+	[ "$status" -ne 0 ]
+	[[ "$output" == *"Operation not permitted"* ]]
```

Comment:
Maybe make the test more strict and look for something like
`*"mkdir:"*"/dev/shm/foo"*"Operation not permitted"*`
in order to not have a false pass when runc merely prints "Operation not permitted" (for whatever reason).

Reply:
Oh, good idea. Changed it for that now, thanks!

Reply:
Also, did the analogous changes to all that can be done the same: `@test "runc run [seccomp] (SCMP_ACT_NOTIFY kill seccompagent)`, `@test "runc run [seccomp] (SCMP_ACT_NOTIFY error chmod)`